xrpl/models/requests/submit_multisigned.py

```diff
@@ -34,17 +30,5 @@ class SubmitMultisigned(Request):
 
     method: RequestMethod = field(default=RequestMethod.SUBMIT_MULTISIGNED, init=False)
     #: This field is required.
-    transaction: Transaction = REQUIRED  # type: ignore
+    tx_json: Transaction = REQUIRED  # type: ignore
```

Comment:
I believe the reason I renamed this was because it felt weird to have a param called `tx_json` that wasn't a JSON-like object.

Reply:
That makes sense. The underlying API takes actual JSON of course, but if this is expecting an instance of the Transaction model then it makes sense to be different.

Reply:
for now let's please leave as is. i know what you guys are saying but at the moment we have a thing going with being 1:1 with the rippled API. can always change